packages/beacon-node/test/unit/monitoring/service.test.ts

```diff
@@ -231,10 +231,9 @@ describe("monitoring / service", () => {
       setTimeout(() => service.stop(), 10);
     });
 
-    function assertError(error: {name?: string; message?: string}): void {
-      expect(logger.error).to.have.been.calledOnce;
-      // errors are not thrown and need to be asserted based on the error log
-      expect(logger.error).to.have.been.calledWithMatch("Failed to send client stats", {}, error);
+    function assertError(error: {message: string}): void {
+      // errors are not thrown and need to be asserted based on the log
+      expect(logger.warn).to.have.been.calledWithMatch("Failed to send client stats", {reason: error.message});
```

Comment:
The assertions here are a bit brittle due to checking the error messages, but should not break unless the log level is changed and having this stricter assertion is important to ensure we are actually testing correctly.